docs/get-started/get-started.md

```diff
@@ -311,6 +313,61 @@ When the user logs out form the app, make sure to run:
 GravatarQuickEditor.logout(Email("{USER_EMAIL}"))
 ```
 
+#### 1.2 Using the provided activity
+
+If using an activity with `android:launchMode="singleTask"` is not an option, you can use the provided activity. With this option, you don't need to modify how your activities are set up.
+
+You need to add the provided activity to your `AndroidManifest.xml`:
+
+```xml
+<activity
+    android:name="com.gravatar.quickeditor.ui.GravatarQuickEditorActivity"
+    tools:node="merge">
+
+    <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+
+        <data
+            android:scheme="https"
+            android:host="yourhost.com"
+            android:pathPrefix="/redirect-url"
+        />
+    </intent-filter>
+</activity>
+```
+
+_Note the important difference here: the `tools:node="merge"` attribute. This is necessary to merge the intent filter with the one defined in the library._
+
+The `GravatarQuickEditorActivity` defines an Activity Result contract that you can use to launch the Quick Editor and handle the result. Here's an example of how you can use it:
+
+```kotlin
+private val getQEResult = registerForActivityResult(GetQuickEditorResult()) { quickEditorResult ->
+        when (quickEditorResult) {
+            GravatarQuickEditorResult.AVATAR_SELECTED -> { ... }
+
+            GravatarQuickEditorResult.DISMISSED -> { ... }
+
+            else -> { ... }
+        }
+}
+
+getQEResult.launch(
+    GravatarQuickEditorActivity.GravatarEditorActivityArguments(
+        GravatarQuickEditorParams { ... },
+        AuthenticationMethod.OAuth(
+            OAuthParams { ... },
+        ),
+    ),
+)
+```
+
+It's important to note that using the `GravatarQuickEditorActivity` you'll only receive the result of the Quick Editor when it's dismissed not instantly as with using the `@Composable` component from your `singleTask` activity.
```

Comment:
I would link to the 1.1 above just in case someone missed it. WDYT?

Reply:
Done! 👍